BetterPartyFinder/Util.cs

```diff
@@ -33,4 +35,138 @@ internal static IEnumerable<World> WorldsOnDataCentre(IPlayerCharacter character
+                break;
+            case "ARC":
+                jobFlags = JobFlags.Archer;
+                break;
+            case "CNJ":
+                jobFlags = JobFlags.Conjurer;
+                break;
+            case "THM":
+                jobFlags = JobFlags.Thaumaturge;
+                break;
+            case "PLD":
+                jobFlags = JobFlags.Paladin;
+                break;
+            case "MNK":
+                jobFlags = JobFlags.Monk;
+                break;
+            case "WAR":
+                jobFlags = JobFlags.Warrior;
+                break;
+            case "DRG":
+                jobFlags = JobFlags.Dragoon;
+                break;
+            case "BRD":
+                jobFlags = JobFlags.Bard;
+                break;
+            case "WHM":
+                jobFlags = JobFlags.WhiteMage;
+                break;
+            case "BLM":
+                jobFlags = JobFlags.BlackMage;
+                break;
+            case "ACN":
+                jobFlags = JobFlags.Arcanist;
+                break;
+            case "SMN":
+                jobFlags = JobFlags.Summoner;
+                break;
+            case "SCH":
+                jobFlags = JobFlags.Scholar;
+                break;
+            case "ROG":
+                jobFlags = JobFlags.Rogue;
+                break;
+            case "NIN":
+                jobFlags = JobFlags.Ninja;
+                break;
+            case "MCH":
+                jobFlags = JobFlags.Machinist;
+                break;
+            case "DRK":
+                jobFlags = JobFlags.DarkKnight;
+                break;
+            case "AST":
+                jobFlags = JobFlags.Astrologian;
+                break;
+            case "SAM":
+                jobFlags = JobFlags.Samurai;
+                break;
+            case "RDM":
+                jobFlags = JobFlags.RedMage;
+                break;
+            case "BLU":
+                jobFlags = JobFlags.BlueMage;
+                break;
+            case "GNB":
+                jobFlags = JobFlags.Gunbreaker;
+                break;
+            case "DNC":
+                jobFlags = JobFlags.Dancer;
+                break;
+            case "RPR":
+                jobFlags = JobFlags.Reaper;
+                break;
+            case "SGE":
+                jobFlags = JobFlags.Sage;
+                break;
+            case "VPR":
+                jobFlags = JobFlags.Viper;
+                break;
+            case "PCT":
+                jobFlags = JobFlags.Pictomancer;
+                break;
+        }
+        return jobFlags;
+    }
+
+    internal static List<JobFlags> GetCurrentPartyJobs()
+    {
+        List<JobFlags> newJobs = new();
+        int plen = new();
+        if (InfoProxyCrossRealm.IsCrossRealmParty())
+        {
+            plen = InfoProxyCrossRealm.GetGroupMemberCount(0);
+            for (uint i = 0; i < plen; i++)
+            {
+                unsafe
```

Comment:
Just make the full function unsafe as all of it calls CS functions, which are by design not really "safe"